config/initializers/sufia.rb

```diff
@@ -105,7 +105,7 @@
   # config.redis_namespace = "sufia"
 
   # Specify the path to the file characterization tool:
-  config.fits_path = "/home/vagrant/fits/fits-0.6.2/fits.sh"
+  config.fits_path = Rails.application.secrets['fits_path']
```

Comment:
This conflicts with the approach taken in a pull request I submitted a little earlier that just installs FITS to /opt/fits.